### ISSUE TITLE
Display all topics in one book, sortable by title or page number

### ISSUE THREAD

**Issue by QuantumTea** [davidrea/indxd#4]
Would be useful to have a display on the individual book pages where you can see all topics in that book sortable by title or by page number.

**Comment by davidrea** [comment]
By clicking into any book, you can view just its topics - is this what you're looking for? It isn't sortable by page right now - but that would be an easy add.

**Comment by QuantumTea** [comment]
Yep, sortable by page number on that view. The other thing I was thinking of was that the topics would be easier to read if they were displayed the same way as the overall topics index.

**Comment by davidrea** [comment]
Maybe I could add some tabbed navigation there to choose between "pill" and list views, with either sortable by topic or page number. A good project for a weekend soon!
